src/index.js

```diff
@@ -20,15 +20,80 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function isEmpty(list) {
+  return !list.length;
+}
+
+function calculate(numList, operList) {
+  const b = numList.pop();
+  const a = numList.pop();
+  const oper = operList.shift();
+  // console.log(oper);
+  if (oper === "+") return a + b;
+  if (oper === "-") return a - b;
+  if (oper === "*") return a * b;
+  if (oper === "/") return Math.round((a / b) * 10) / 10;
+  return 0;
+}
+
+function render(currentNum, numberArr, operatorArr) {
+  const numberList = numberArr;
+  const operatorList = operatorArr;
+  const number = currentNum;
+  var operUsed = false;
+
+  const numberBtnList = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+  const operatorBtnList = ["+", "-", "*", "/", "="];
+
+  const numberClick = (ClickedNumber) => {
+    if (operUsed === false) {
+      const newNumber = number * 10 + ClickedNumber;
+      render(newNumber, numberList, operatorList);
+    } else {
+      render(ClickedNumber, numberList, operatorList);
+    }
+  };
```

Comment:
```suggestion
    if (operUsed ) {
    render(ClickedNumber, numberList, operatorList);
    return;
    }

    const newNumber = number * 10 + ClickedNumber;
    render(newNumber, numberList, operatorList);
  };
```

`if ~ else` 사용하시는걸 지양하시는게 좋아요.

조건이 복잡해질수록 거의 읽기 불가능한 코드를 작성할 확률이 높아요.

위와 같은 경우에는 예시처럼 guard clauses 방식으로 처리하시는게 좋아요.